__tests__/endpoints/topics.test.js

```diff
@@ -0,0 +1,24 @@
+const { app, data, seed, db, request } = require("../../testImports");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+	test("GET: 200 sends an array of topics to the client", () => {
+		return request(app)
+			.get("/api/topics?sort")
+			.expect(200)
+			.then(({ body: { topics } }) => {
+				expect(topics.length).toBe(3);
+				topics.forEach((topic) => {
+					expect(typeof topic.slug).toBe("string");
+					expect(typeof topic.description).toBe("string");
+				});
```

Comment:
Solid tests, great that you're checking the length of the array to avoid false positives if passed an empty array
Could use the `toMatchObject` matcher to make it a bit more succinct, but this does the job well!